### PR TITLE
Update back button integration docs

### PR DESCRIPTION
**Pull request by jhesgodi**
Added some extra details to the back button integration section. This will provide a bit more guidance on how to do the integration.

### CODE REVIEW

File: docs/redux-integration.md [hunk 1]
```diff
@@ -148,12 +148,23 @@ By using the following snippet, your nav component will be aware of the back but
...
 import { BackHandler } from "react-native";
 import { NavigationActions } from "react-navigation";
 
 /* your other setup code here! this is not a runnable snippet */
 
-class ReduxNavigation extends React.Component {
+const Navigator = createStackNavigator(AppRouteConfigs);
+const AppNavigator = reduxifyNavigator(Navigator, "root");
+
+class  extends React.Component {
```
Comment (Ashoat) [inline docs/redux-integration.md]: Can you add the class name `ReduxNavigation` back here?

File: docs/redux-integration.md [hunk 2]
```diff
@@ -148,12 +148,23 @@ By using the following snippet, your nav component will be aware of the back but
 
 ```es6
 import React from "react";
+import {
+  reduxifyNavigator,
+  createReactNavigationReduxMiddleware,
```
Comment (Ashoat) [inline docs/redux-integration.md]: You can skip the second and third imports here since they're not used.

**Comment by Ashoat** [comment]
Ping!

**Comment by brentvatne** [comment]
hey @jhesgodi if you're not interested in updating the PR just let me know and close the issue :) thanks!

**Pull request closed without merging**